src/App.jsx

```diff
@@ -9,36 +11,18 @@ export const tabs = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">Selected tab is Tab 1</h1>
-
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">
-              Tab 1
-            </a>
-          </li>
-
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">
-              Tab 2
-            </a>
-          </li>
+export const App = () => {
+  const [selectedTab, onTabSelected] = useState(tabs[0]);
```

Comment:
Please, rename `onTabSelected` into `setSelectedTab`. It's said that we need to pass the `onTabSelected` callback to the `Tabs` as a props, but no need to name `selectTab`'s setter somehow other. It just decreases code readability and clarity